src/ProjectConfiguration.ts

```diff
@@ -22,6 +23,7 @@ export interface BaseProjectConfiguration {
   appUUID: string;
   requestedPermissions: string[];
   i18n: LocalesConfig;
+  fallbackLocale: string;
```

Comment:
Just because we can doesn't necessarily mean we should. Like you said, if we have a special-case field in the locales config object then we have to remember to handle the special case everywhere.